modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java

```diff
@@ -394,6 +395,10 @@ public void register(Registry registry) {
 			"4.1.0", "4.2.0",
 			new com.liferay.dynamic.data.mapping.internal.upgrade.v4_2_0.
 				UpgradeDDMFormInstanceRecord());
+
+		registry.register(
+			"4.2.0", "4.3.0",
+			new UpgradeDDMFieldAttributeFromDLFileEntryType());
```

Comment:
Better rename to UpgradeDLFileEntryTypeDDMFieldAttribute

Reply:
thanks!